zerver/openapi/zulip.yaml

```diff
@@ -3107,26 +3107,19 @@ paths:
                                 set of configured [linkifiers](/help/add-a-custom-linkifier)
                                 for the organization has changed.
 
-                                Processing this event is important to doing Markdown local echo
+                                Processing this event is important for doing Markdown local echo
                                 correctly.
 
-                                The client will not receive this event unless the event queue
-                                is registered with `linkifier_url_template` client capability set to `true`.
+                                **Changes**: As of Zulip 7.0 (feature level 176), clients will not
+                                receive this event unless the event queue is registered with the
+                                client capability `{"linkifier_url_template": true}`. This is because
```

Comment:
It looks like with this commit, there's an important piece of information about the current API that only appears when reading the "Changes" notes: namely, that you won't get this event unless you announce that client capability.

In general, I think it's good for us to maintain a principle that you only need to read "Changes" notes when you're concerned with supporting older versions — that if you skip all the "Changes" notes, you get a coherent and self-contained description of the current API.

That's good for people writing clients that truly don't need to care about older versions (which is probably most clients — any ad-hoc bot, if the place it's meant to be deployed is either Zulip Cloud or another server that's kept up to date); and I think it's also helpful even when working on the full-scale clients that do care about older versions, because it lets the reader start with one coherent picture and then add complications.

So I think the old version of this paragraph, and the next one with a link about client capabilities, worked pretty well. The old version of the "Changes" note was a bit awkwardly duplicative; I think one good fix for that would be to turn it around, so that it says something like:

> Before Zulip 7.0 (feature level 176), the `linkifier_url_template` client capability was not required, and the linkifiers contained a URL format string instead of a URL template.